docs/source/contributing.rst

```diff
@@ -2,49 +2,43 @@
 Contributing to GTG
 ===================
 
-GTG uses Bazaar_ for versioning. It might be useful to read `Bazaar's tutorial`_ first.
+GTG uses GitHub_ for versioning. It might be useful to read `GitHub's Guide`_ first.
```

Comment:
We use git, not github for versioning :) Also add somewhere a link to this useful tutorial about git: http://learnxinyminutes.com/docs/git/